.github/workflows/release.yaml

```diff
@@ -68,14 +68,16 @@ jobs:
           go test ./charts/redpanda -update -short
 
       # Commit changes must be run before chart-releaser execution, because auto commit would include helm chart tar.gz artifact
-      - name: Commit changes for testdata golden files
-        run: |
-          git config --global user.name 'vbotbuildovich'
-          git config --global user.email 'vbotbuildovich@users.noreply.github.com'
-          git config --global --add --bool push.autoSetupRemote true
-          git add --all
-          git commit -m "[Auto Commit] Update testdata golden files"
-          git push
+      # TODO: Find a solution for protection ruleset that prevents from pushing to main branch any commit that do not pass
```

Comment:
Can we file a github issue for this and link to it from this TODO?

Might be nice to make an action that turns TODO's into Issues 🤔

Reply:
The ticket is created in our internal JIRA.